playbooks/openstack/README.md

```diff
@@ -71,8 +71,8 @@ The OpenStack environment must satisfy these requirements:
 * The keypair for SSH must be available in OpenStack
 * You must have a`keystonerc` file that lets you talk to the OpenStack services
 * In order to install an OpenShift cluster and deploy services, we recommend
-that you have a minimum of 30 security groups, 200 security group rules, and 200
-ports available in your quota.
+that you have a minimum of 100 security groups, 500 security group rules, and
+500 ports available in your quota.
```

Comment:
Is this recommendation for all deployments, or specifically for kuryr? I wonder if we may want to separate the two, since I think 100 security groups is far over the default setting for vanilla deployments of openstack.

Reply:
Good point! I'll make the differentiation